data/json/furniture_and_terrain/furniture-fungal.json

```diff
@@ -48,5 +48,18 @@
     "required_str": -1,
     "flags": [ "TRANSPARENT", "CONTAINER", "SEALED", "ALLOW_FIELD_EFFECT", "FLAMMABLE_ASH", "FUNGUS", "MOUNTABLE", "SHORT" ],
     "bash": { "str_min": 6, "str_max": 20, "sound": "poof.", "sound_fail": "poof." }
+  },
+  {
+    "type": "furniture",
+    "id": "f_fungal_tangle",
+    "name": "fungal tangle",
+    "description": "Thick, ropy tendrils of fungus have risen from the ground and gathered into an impenetrable clump.",
+    "looks_like": "mon_fungal_wall",
+    "symbol": "&",
+    "bgcolor": "light_gray",
+    "move_cost_mod": -1,
+    "required_str": -1,
+    "flags": [ "ALLOW_FIELD_EFFECT", "FLAMMABLE_ASH", "FUNGUS" ],
```

Comment:
Unless I'm reading code wrong, `ALLOW_FIELD_EFFECT` flag is meant to work only with or without the `SEALED` flag. Alone `ALLOW_FIELD_EFFECT` flag won't do anything useful.

Reply:
Woops, thanks for spotting. I just forgot to delete it after copying the code from fungal clump